### DISCUSSION TITLE
New Driver made with Berry for Eurom Pac Wifi

### DISCUSSION THREAD

**Discussion by jriguera** [arendst/Tasmota#21739] [Show and tell]
Regarding the topic in the discussion https://github.com/arendst/Tasmota/discussions/21653, I followed the suggestion and I implemented a Berry driver https://github.com/jriguera/tasmota_eurom_pac_wifi/tree/main/esp32  for  Eurom Pac Wifi.  Additional Home Assistant configuration and screenshot are also provided in the repository

Thanks all for your support!

**Comment by sfromis** [comment]
Looks pretty good as a starting project with Berry :smile:
